### PR TITLE
add sort on id, name, status, dateonwaitinglist

### PR DESCRIPTION
**Pull request by ashleygwilliams**

### CODE REVIEW

**Comment by ashleygwilliams** [comment]
@steveklabnik r?

File: app/components/sorting-key.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  tagName: 'th',
+  classNameBindings: ['isSorted:active', 'isAsc:asc', 'isDesc:desc'],
+  isSorted: (function() {
```
Comment (steveklabnik) [inline app/components/sorting-key.js]: https://github.com/artisan-tattoo/assistant-frontend/pull/54/files#diff-d8fbe1b980499a71bd6949c7876eb1d5R16 this style is considered better

Comment (steveklabnik) [inline app/components/sorting-key.js]: http://emberjs.com/api/classes/Ember.computed.html

**Comment by steveklabnik** [comment]
:confetti_ball:

**Pull request merged**